cip-0008.md

```diff
@@ -0,0 +1,56 @@
+<pre>
+  CIP: 8
+  Title: CIP Implementation Bounties
+  Author: Robby Dermody <robbyd@counterparty.io>
+  Status: Draft
+  Type: Process
+  Created: 2017-02-23
+</pre>
+
+## Abstract ##
+
+This CIP outlines the process by which bounties for work on Counterparty or Counterparty-related software should be proposed, created, operated and monitored.
+
+## Motivation ##
+
+Implementation of most CIPs requires a non-trivial amount of highly skilled design and development time. Bounties raised from the Counterparty community can be a useful tool to promote implementation of specific CIPs via raising the funds sometimes necessary to incentivize talented developers and compensate them for their time. This is especially true given the non-profit, volunteer nature of Counterparty development in general.
+
+## Rationale ##
+
+Prior to this process, implementation of CIPs via bounties was performed somewhat ad-hoc. This raised some confusion and exposed several ambiguities, such as potential mismatches in expectations between the individuals donating to the effort and the parties implementing the effort, as well as confusion regarding the degree of latitude to which the development party and/or project maintainers could exercise creative control of the implementation.
+
+The goal of this CIP is to remove these and other potential points of conflict and institute a process that maximizes the chance of success while remaining as lightweight as possible.
+
+## Specification ##
+
+### CIP Custodians ###
+
+The Counterparty Foundation Board may appoint one or more CIP Custodians, who shall be in charge of handling funds for currently open CIP bounty efforts. CIP custodians shall serve for an indeterminate amount of time and may be freely appointed and removed by the Counterparty Foundation members via a majority vote.
+
+In its fiduciary duty, the Board shall seek to only appoint individuals that have a both a proven track record of involvement in Counterparty, as well as demonstrate trustworthy character traits. Each CIP custodian may not serve as a Counterparty Foundation board member at the same time, and may not serve as both a CIP author and CIP custodian for any specific proposal.
```

Comment:
How about:

A CIP custodian may not simultaneously serve as Counterparty Foundation board member.  The CIP author may not serve as a CIP custodian for any specific proposal.